src/EditorFeatures/Core/Implementation/Suggestions/SuggestedActions/SuggestedAction.cs

```diff
@@ -251,8 +251,7 @@ ImageMoniker ISuggestedAction.IconMoniker
                 {
                     foreach (var service in SourceProvider.ImageMonikerServices)
                     {
-                        if (service.Value.TryGetImageMoniker(tags, out var moniker) &&
-                            !moniker.IsNullImage())
+                        if (service.Value.TryGetImageMoniker(tags, out var moniker) && !moniker.Equals(default(ImageMoniker)))
```

Comment:
I wasn't going to move all of the SuggestedAction stuff out purely for IsNullImage which lives in the Imaging assembly. We already presume in other places that IsNullImage is default(ImageMoniker), so I didn't feel bad about this.